### PR TITLE
Add a Symfony integration test

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add a Symfony integration test. See after the table for more informations
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | CI is green
| Possible impacts? | 

This repository contains a Bundle, that provides needed configuration items to be integrated into a Symfony application: there is two Compiler Passes, a standard Configuration file, a standard Extension file, a YAML services definition file...

If today we merge changes that modify the above files and make them invalid, the CI will not detect this.

This test aims to fix this.

This test builds a very small Symfony application, as small as possible, with as few configuration as possible, and simply add the TranslationToolsBundle to it then boot it. If we merge changes that modify the above files and make them invalid, this test will fail. It's very simple. This tests simply verifies "this Bundle can be added to a Symfony application, this will not crash".

### CODE REVIEW

**Comment by matks** [comment]
This PR only tests whether the bundle can be added to a Symfony 4 application. I guess a 2nd step would be to verify it works with Symfony 3 and 5.

**Review by mickaelandrieu — COMMENTED** [review]
Some comments, I _believe_ Symfony will throw an exception in this case if the ErrorHandler is set the right way.

File: Tests/SymfonyIntegration/run.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+require_once __DIR__ . '/../bootstrap.php';
+require_once __DIR__ . '/AppKernel.php';
+
+use PHPUnit\Framework\Assert as Assert;
+use PrestaShop\TranslationToolsBundle\Translation\Compiler\Smarty\TranslationTemplateCompiler;
+
+$kernel = new AppKernel('test', false);
```
[the remark this answers is not in the document]
Comment (matks) [reply Tests/SymfonyIntegration/run.php:9]: I can't figure it out. I tried many Monolog configurations, but I fail to output deprecation warnings to stdout

File: Tests/SymfonyIntegration/AppKernel.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function registerBundles()
+    {
+        date_default_timezone_set('UTC');
+
+        $bundles = [
+            new Symfony\Bundle\FrameworkBundle\FrameworkBundle(),
```
[the remark this answers is not in the document]
Comment (matks) [reply Tests/SymfonyIntegration/AppKernel.php:19]: Apparently it's not needed, I tried.

File: Tests/SymfonyIntegration/run.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (mickaelandrieu) [reply Tests/SymfonyIntegration/run.php:9]: On my way, it's my coffee break ☕ 

Let's see if I'm still good 🤣

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks

**Pull request merged**